test/unit/Ramm/setup.js

```diff
@@ -10,13 +10,6 @@ const { parseEther } = ethers.utils;
 const SPOT_PRICE_A = parseEther('0.0347');
 const SPOT_PRICE_B = parseEther('0.0152');
 
-async function getState(ramm) {
-  const { nxmReserveA: nxmA, nxmReserveB: nxmB } = await ramm.slot0();
-  const { ethReserve: eth, budget, updatedAt: timestamp } = await ramm.slot1();
-  const ratchetSpeed = await ramm.ratchetSpeed();
-  return { nxmA, nxmB, eth, budget, ratchetSpeed, timestamp };
-}
-
```

Comment:
This helper function was written before we wrote `ramm.loadState`. Removing in favour of `loadState` for consistency.